config/configgrpc/configgrpc.go

```diff
@@ -61,8 +61,8 @@ type ClientConfig struct {
 	// The compression key for supported compression types within collector.
 	Compression configcompression.Type `mapstructure:"compression"`
 
-	// TLSSetting struct exposes TLS client configuration.
-	TLSSetting configtls.TLSClientSetting `mapstructure:"tls"`
+	// Config struct exposes TLS client configuration.
+	Config configtls.ClientConfig `mapstructure:"tls"`
```

Comment:
Could you leave in the old field name while doing this, but leave a Godoc comment that marks it as deprecated in the upcoming version (v0.95.0)?

Here's an example of what this looks like: https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/configgrpc.go#L51

The same applies for the other renamed types/fields.

Reply:
Ohh wow. Didn't know you could do this (copy types).
Thanks for the example.